R/snapshot.R

```diff
@@ -681,7 +681,12 @@ renv_snapshot_library_diagnose_missing_description <- function(library, pkgs) {
 renv_snapshot_description <- function(path = NULL, package = NULL) {
 
   # read DESCRIPTION file
-  path <- path %||% renv_package_find(package)
+  path <- path %||% {
+    path <- renv_package_find(package)
+    if (!nzchar(path))
+      stopf("package '%s' is not installed", package)
+  }
+
```

Comment:
Do you want to check the path exists even if it's user supplied?

Reply:
Good catch!